science/veryfasttree/Portfile

```diff
@@ -0,0 +1,44 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           cmake 1.1
+PortGroup           github 1.0
+
+github.setup        citius veryfasttree 4.0.3 v
+github.tarball_from archive
+revision            0
+
+categories          science
+license             GPL-3
+platforms           darwin
```

Comment:
this is the default since MacPorts v2.7 and can thus be removed

Reply:
platforms removed, done